frontend/tasks/content.js

```diff
@@ -33,6 +33,11 @@ gulp.task('content-build-en', () => {
     .pipe(gulp.dest('./locales/en'));
 });
 
+gulp.task('content-build-de', () => {
+  return gulp.src('./locales/de/**/*')
+    .pipe(gulp.dest('./locales/de-DE'));
+});
+
```

Comment:
Do we need a matching `content-watch-de` task? Or is "en" a special special case?